data/mods/Dark-Skies-Above/monsters/alien_cyborgs.json

```
@@ -286,5 +286,34 @@
     "anger_triggers": [ "FRIEND_ATTACKED", "HURT" ],
     "death_function": [ "NORMAL" ],
     "flags": [ "SEES", "HEARS", "SMELLS", "BASHES", "DESTROYS", "ACIDPROOF", "CAN_DIG", "ARTHROPOD_BLOOD" ]
+  },
+  {
+    "id": "mon_dks_skirmsoldier",
+    "type": "MONSTER",
+    "name": { "str": "alien skirmisher" },
+    "description": "A slouched figure, slightly smaller than an average adult human.  Beneath layers of full-body combat armor and a full suite of cybernetic modifications are glimpses of rubbery, abscessed flesh.  It is capable of short, boosted jumps using a device on its back, closing gaps to cause havoc with its wicked integrated melee weapons.  Severed organic parts dangle from its equipment, morbid trophies from past battles.",
+    "default_faction": "invader_alien",
+    "bodytype": "human",
+    "categories": [ "ALIEN" ],
+    "species": [ "ROBOT" ],
+    "volume": "52500 ml",
+    "weight": "71500 g",
+    "hp": 45,
+    "speed": 90,
+    "material": [ "steel" ],
+    "symbol": "$",
+    "aggression": 100,
+    "morale": 100,
+    "color": "light_gray",
+    "dodge": 1,
+    "armor_bash": 15,
```

```suggestion
    "armor_bash": 25,
```
I'd like to make it hard to engage with crushing weapons, because the current weapons meta favors crushing or piercing weapons too much.